src/measure_theory/measure/measure_space.lean

```diff
@@ -3599,6 +3599,10 @@ e.measurable_embedding.restrict_map _ _
 lemma map_ae (f : α ≃ᵐ β) (μ : measure α) : filter.map f μ.ae = (map f μ).ae :=
 by { ext s, simp_rw [mem_map, mem_ae_iff, ← preimage_compl, f.map_apply] }
 
+lemma quasi_measure_preserving_symm (μ : measure α) (e : α ≃ᵐ β) :
+  quasi_measure_preserving e.symm (map e μ) μ :=
+⟨e.symm.measurable, by rw [measure.map_map, e.symm_comp_self, measure.map_id]; measurability⟩
```

Comment:
can you add the missing bits to get the above proof working here also?

Reply:
Are you sure that's true?
```
lemma symm (e : α ≃ᵐ β) (h : quasi_measure_preserving e μa μb) :
  quasi_measure_preserving e.symm μb μa :=
⟨e.symm.measurable,
  begin
    have := h.absolutely_continuous.map e.symm.measurable,
    rw [map_map e.symm.measurable e.measurable, e.symm_comp_self, map_id] at this,
    sorry
    /-
    α: Type u_1
    β: Type u_2
    m0: measurable_space α
    _inst_1: measurable_space β
    μa: measure α
    μb: measure β
    e: α ≃ᵐ β
    h: quasi_measure_preserving ⇑e μa μb
    this: μa ≪ map ⇑(e.symm) μb
    ⊢ map ⇑(e.symm) μb ≪ μa
    -/
  end⟩
```

Reply:
Definitely not. The definition I have in my head is not the current Lean one...